PullRequestMonitor/View/SingleProjectView.xaml

```diff
@@ -21,5 +22,13 @@
                 <local:PullRequestListView Margin="5" DataContext="{Binding Approved}"/>
             </StackPanel>
         </ScrollViewer>
+        <ScrollViewer Grid.Row="2" VerticalScrollBarVisibility="Auto">
+            <Expander IsExpanded="True">
```

Comment:
This needs to move into the other ScrollViewer otherwise the scroll bar doesn't appear when the active pull requests overflow the window space (and then you can't see the completed pull requests at all).